en/docs/observability/observability-overview.md

```diff
@@ -0,0 +1,71 @@
+
+You can expand the graph by clicking and dragging the cursor over the period you want to drill down.
+
+You can view the Choreo service logs in the **Logs**  view below the graph. Clicking on either graph will update the **Logs** view to contain the corresponding log entries generated at that time. You can use these logs to identify any evident reasons for any latency and throughput anomalies you detect using the graph. 
+
+ ![Logs View](../assets/img/observability/logs.png){: style="height:55%;width:70%"} 
+
+### Diagnostics View
+
+![Diagnostic-view](../assets/img/observability/diagnostic-view.png){: style="height:55%;width:70%"}
+
+The **Diagnostics view** provides the capability to view errors, throughput, latencies, CPU usage, memory usage, and logs simultaneously for a particular event. You can detect and analyze errors and anomalies in detail via the **Diagnostics view**.
+
+By default, the time range selected for the **Throughput & Latency** graphs is the same time range used for the **Diagnostics View**. 
+
+A **bin** is a horizontal section of the graph for a particular period. A bin consists of the following:
+
+- **Start time and end time** - Denoted by the dotted lines separating each bin.
+- **Logs** - Log events that occurred during the bin's time range appear in the **Logs** column, along with the timestamp the log entries started to appear and the log count. These log entries appear according to precedence. The error logs are listed first, followed by the info logs. Each bin shows a maximum of five log entries sorted based on the log count.
+- **Error** - Number of HTTP errors occurred at the selected time.
+- **TP** - Throughput of the requests at the selected time (req/s).  
+- **Latency** - Latency of the request at the selected time (ms).
+- **CPU** - CPU usage at the selected time (millicores).
+- **Memory** - Memory usage at the selected time (MiB).
+
+
+### Trace Requests
+
+You can trace requests received by your service at a given timestamp by clicking on a point in the time axis of either the throughput or the latency graph.  The low-code view shows the received requests, along with each of its request time, latencies, and its status. Furthermore, you can view the status code and the request latency when you click on a particular request.
+
+The low-code diagram will also display the average request latency and the success rate for each client request corresponding to the time selected from the graphs. 
+
+
+## View and Filter All Logs
+
+![Logs Panel](../assets/img/observability/logs-panel.png){: style="height:45%;width:70%"}
+
+The **Logs** panel includes all logs generated by the service. You can filter it based on time or based on text. 
+
+You can also download the log entries using the download button. It will save the log entries as a .zip file in your preferred location.
```

Comment:
Suggestion: "To download the log entries, click **Download**. As a result, the log entries are saved as a .zip file in the location you specify."

Changing this as Microsoft Style guide recommends to avoid using the word "button".

Reply:
changed.